Follow-up to #5083: Minor Doc Fixes

HereAround opened the pull request:
* Add `# Examples` to all FTheoryTools docstring examples.
* Enforce imperative in doc strings by renaming `Returns` to `Return` (and friends) in doc strings.

cc @apturner @emikelsons

HereAround committed:
FTheoryTools: Add back # Examples to the doc strings

HereAround committed:
[FTheoryTools] Returns -> Return in doc strings

HereAround commented:
@lgoettgens As discussed last week, this is a follow-up to https://github.com/oscar-system/Oscar.jl/pull/5083. Please consider merging if you are satisfied.

lgoettgens commented:
Could you also check for words like `Checks`, `Creates`, `Computes`, and `Constructs` as the first word of a docstring. Those should also lose the `s`, analogous to `Returns`.

lgoettgens commented on experimental/FTheoryTools/src/FamilyOfG4Fluxes/attributes.jl:
```diff
@@ -150,12 +154,13 @@ to be a non-negative integer.
 The family of fluxes is given as a linear combination of cohomology classes with 
 integral (denoted by ``a_i``) and rational (denoted by ``r_i``) coefficients. This
-method returns the quadratic polynomial in these coefficients that encodes the
+method Return the quadratic polynomial in these coefficients that encodes the
```
Faulty change

HereAround replied:
Fixed.

lgoettgens commented on experimental/FTheoryTools/src/FamilyOfG4Fluxes/properties.jl:
```diff
@@ -9,9 +9,10 @@ Performs elementary necessary (but not sufficient) tests to check if the
 
 If any test fails, the family is definitely not well-quantized and the method 
-returns `false`. If all tests pass, it returns `true`, indicating the fluxes 
+Return `false`. If all tests pass, it Return `true`, indicating the fluxes 
```
Two faulty changes

HereAround replied:
Fixed.

lgoettgens commented on experimental/FTheoryTools/src/G4Fluxes/properties.jl:
```diff
@@ -20,8 +20,9 @@ for all pairs of toric divisors ``D_i``, ``D_j`` in the ambient variety, where `
 restricts to ``c_2(\widehat{Y}_4)`` on the hypersurface.
 
-If all these integrals evaluate to integers, this method returns `true`; otherwise, it returns `false`.
+If all these integrals evaluate to integers, this method Return `true`; otherwise, it Return `false`.
```
Two faulty changes

HereAround replied:
Fixed.

HereAround committed:
[FTheoryTools] Checks -> Check in doc strings

HereAround committed:
[FTheoryTools] Creates -> Create in doc strings

HereAround committed:
[FTheoryTools] Computes -> Compute in doc strings

HereAround committed:
[FTheoryTools] Constructs -> Construct in doc strings

HereAround committed:
[FTheoryTools] Constructs -> Construct in two comments

HereAround committed:
[FTheoryTools] Adds -> Add in doc strings

HereAround commented:
> Could you also check for words like `Checks`, `Creates`, `Computes`, and `Constructs` as the first word of a docstring. Those should also lose the `s`, analogous to `Returns`.

I checked and modified all of these. In additions found `Adds` which I changed accordingly. I could not find anything else at the top of my head (`Sets`, `Reads`, `Verifies`, ...), so hopefully this is complete.

lgoettgens reviewed (approved):
Thanks! I didn't check that you got all cases, but the cases you did change look good.
